### PR TITLE
first pass at collecting command output for demo

### PR DESCRIPTION
**Pull request by vjeffrey**
heya peoples can you take a look and let me know what you think of this??  we would want the files to actually be saved somewhere in the www/ folder that the demo branch will introduce and then pull them in with the html....so i would need to make sure this overwrites too -- but i wanted to make sure something like this would be ok??. thank ya :)

### CODE REVIEW

File: tasks/get_command_output.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+require 'open3'
```
Comment (chris-rock) [inline tasks/get_command_output.rb]: Lets use https://github.com/chef/mixlib-shellout or 

```
backend = Train.create('local', target_config)
# start or reuse a connection
conn = backend.connection
# loop around commands
cmd = conn.run_command('inspec version')
cmd.stdout
conn.close
```

under the hood, train is using mixlib-shellout

File: tasks/get_command_output.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
```
Comment (chris-rock) [inline tasks/get_command_output.rb]: should we name the command like `inspec_simulator`?

File: tasks/get_command_output.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+require 'open3'
+
+commands = { 'inspec_exec' => 'inspec exec examples/profile/controls/example.rb', 'inspec_version' => 'inspec version' }
+
+commands.each do |keyname, command|
+  stdout, _stdeerr, _status = Open3.capture3(command)
+
+  # filename should be key.txt...and we should be saving it somewhere in www/
+  out_file = File.new("#{keyname}.txt", 'w')
```
Comment (chris-rock) [inline tasks/get_command_output.rb]: use a base path as parameter, but this looks great

File: Rakefile [hunk 4]
```diff
@@ -17,6 +17,13 @@ end
 desc 'Run robocop linter'
 task lint: [:rubocop]
 
+# update command output for demo
+desc 'Running commands and saving TBD'
```
Comment (chris-rock) [inline Rakefile]: +1

**Comment by chris-rock** [comment]
@vj this looks great, lets just switch the libs, running it via train would allow us build a predefined container that is including the latest inspec later. This would make results consistent across operating systems and CI pipelines.

**Comment by chris-rock** [comment]
Once we have the basic commands covered, LGTM

**Comment by vjeffrey** [comment]
@chris-rock updated to use train

**Comment by chris-rock** [comment]
Thanks @vjeffrey for this great improvement! I'll create a ticket for another iteration :-)

File: tasks/command_simulator.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+commands = { 'inspec_exec' => 'inspec exec examples/profile/controls/example.rb', 'inspec_version' => 'inspec version' }
+
+commands.each do |keyname, command|
+  backend = Train.create('local')
+  conn = backend.connection
+  # loop around commands
+  cmd = conn.run_command(command)
+  cmd.stdout
+  conn.close
```
Comment (chris-rock) [inline tasks/command_simulator.rb:12]: move 

```
backend = Train.create('local')
conn = backend.connection
```

and `conn.close` out of the loop

**Pull request merged**